stories/dynamic-side-content/dynamic-side-content.stories.js

```diff
@@ -0,0 +1,157 @@
+import '../../dist/dynamic-side-content.css';
+
+export default {
+    title: 'Layouts/Dynamic Side Content',
+    parameters: {
+        description: `
+Dynamic side content is a layout control that displays supplemental content in a separate area to support the users' understanding of the main content displayed.
+
+## The dynamic side content consists of two elements:
+*  \`.fd-dynamic-side__side\`: Side content section.
+*  \`.fd-dynamic-side__main\`: Main content section.
+
+
+## Usage
+### Use dynamic side content if:
+*  You want to display information that:
+  *  Will enrich the main content and will help the user better perform his/her tasks;
+  *  Only makes sense when displayed next to the main container (side-by-side);
+  *  Influences the main content (for example, a filter for list; settings for chart, details for map).
+*  Users should have access to all of the key functions and critical information in the app even if they do not see the side content.  
+This is important because on smaller screen sizes it may be difficult to display the side content in a way that is easily accessible for the user.  
+
+
+## Sizes
+The screen width ratio between the side and main content area will vary depending on the available screen space or window size. The default style displays side content on the left or right side of the main content area but if the space is limited, it can display at the bottom.
+
+| Screen size &nbsp;&nbsp; | Main content width ratio &nbsp;&nbsp; | Side content width ratio &nbsp;&nbsp; | Modifier class &nbsp;&nbsp; |
+| :--- |:--- | :--- | :--- |
+| >= 1400px &nbsp;&nbsp; | 75% | 25% | \`fd-dynamic-side--xl\` |
+| < 1400px & > 720px &nbsp;&nbsp; | 66.66% | 33.33% | \`fd-dynamic-side--md\` |
+| < 720px &nbsp;&nbsp; | 100% | Disappears | \`fd-dynamic-side--sm\` |
+| > 720px Equal split mode &nbsp;&nbsp; | 50% | 50% |
```

Comment:
Write it like this: Equal split mode (> 720px)

Reply:
done